lighthouse/blueprints/plates.py

```diff
@@ -26,10 +26,14 @@ def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
         samples = get_positive_samples(barcode)
 
         if not samples:
-            return {"errors": ["No samples for this barcode"]}, HTTPStatus.BAD_REQUEST
+            return {"errors": ["No samples for this barcode: " + barcode]}, HTTPStatus.BAD_REQUEST
 
         # add COG barcodes to samples
-        centre_prefix = add_cog_barcodes(samples)
+        try:
```

Comment:
Does adding the cog barcodes ever fail?

Reply:
It was failing for me locally, as the `add_cog_barcodes` method can throw an Exception. So guess its possible but not sure how likely it is in UAT/PROD..